### DISCUSSION TITLE
Temporary zoom in a single panel

### DISCUSSION THREAD

**Discussion by stefano-cherchi** [grafana/grafana#54695] [Dashboards, Panels & Visualizations]
Hi all, 

the default behaviour in Grafana, when you click&drag to select a segment of a chart, is to re-set the time window of the entire dashboard to the selection. 

This means that all the queries of the dashboard must be executed anew. In case of complex dashboards with multiple panels, widget, and tables, this can result in slow performance and unnecessary load on the database. 

Sometimes, as an operator I need to zoom into a segment of just one metric to take a quick peek at an eye-catching feature or determine the time of an event in more detail, etc, and then zoom back to the "whole picture" without affecting the rest of the dashboard. 

Other products come with this feature out-of-the-box (I'm thinking of Splunk), but I couldn't find it in Grafana or in its plugin library. 

Is there a way to activate this feature, or is it in the roadmap?

Regards

S

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
